editor.html

```diff
@@ -375,6 +375,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                                 <li><span title="Croatian"><a lang="hr" href="#" class="action lang-choice" id="hr" tabindex="124">Hrvatski</a></span></li>
                                 <li><span title="Chinese (simplified)"><a lang="zh-CN" href="#" class="action lang-choice" id="zh-CN" tabindex="125">简体中文</a></span></li>
                                 <li><span title="Chinese (Hong Kong)"><a lang="zh-HK" href="#" class="action lang-choice" id="zh-HK" tabindex="126">繁体中文</a></span></li>
+                                <li><span title="Chinese (Taiwan)"><a lang="zh-TW" href="#" class="action lang-choice" id="zh-TW" tabindex="127">Chinese (Taiwan)</a></span></li>
```

Comment:
This should be in the original language.

Reply:
This string has been added in this PR as part of adding the language https://github.com/bbcmicrobit/PythonEditor/pull/316/files#diff-0a35fcc3a72c9571d31502ed16d78d4aR147 so it's not as yet been translated.

Reply:
I think having the right translation in the menu is an important i18n factor, and as mentioned in https://github.com/bbcmicrobit/PythonEditor/pull/316#discussion_r366774276, since we are merging to master I think we should wait for the translation to be finished to be able to include this.

Reply:
This has now been updated

Reply:
Can we ask the translator to use the same format as the other chinise translations?
![image](https://user-images.githubusercontent.com/29712657/72444324-bc832980-37a7-11ea-962a-80ae3cd2f3f7.png)

Reply:
For the menu, Chinese (simplified) and Chinese (traditional) were copied from MakeCode. They only have one Chinese traditional project live, so I expect they do not need to signify the localisation. 

I expect the current translation and format of the strings for `zh-TW` is correct and that we should ask for the` zh-HK` to follow the same format.

Reply:
> I expect the current translation and format of the strings for zh-TW is correct and that we should ask for the zh-HK to follow the same format.

Do you mean to ask zh-HK to change from `繁体中文` to something like `中文（XX)`?

Reply:
Yes. I have done this in https://github.com/bbcmicrobit/PythonEditor/pull/316/commits/2c3f20f2a310959b2260ed527cde1e96c8c56e03 and verified it is correct with Waris.

Reply:
Great, thanks!